Kudu.Core/Infrastructure/FunctionAppHelper.cs

```diff
@@ -1,10 +1,16 @@
-﻿namespace Kudu.Core.Infrastructure
+﻿using System.IO;
+
+namespace Kudu.Core.Infrastructure
 {
     internal static class FunctionAppHelper
     {
-        public static bool LooksLikeFunctionApp()
+        const string hostJson = "host.json";
+        public static bool LooksLikeFunctionApp(string projectFolder)
```

Comment:
both, since we are checking for "host.json" and environment variable, not `.sln` `csproj` or `index.js`